--- gtk+-3.13.3/gtk/Makefile.in
+++ gtk+-3.13.3/gtk/Makefile.in
@@ -114,13 +114,13 @@
 @USE_WIN32_TRUE@am__append_12 = $(gtk_use_win32_private_h_sources)
 @USE_QUARTZ_TRUE@am__append_13 = $(gtk_use_quartz_c_sources)
 @USE_QUARTZ_TRUE@am__append_14 = $(gtk_use_quartz_private_h_sources)
 @USE_QUARTZ_FALSE@@USE_WIN32_FALSE@@USE_X11_FALSE@am__append_15 = $(gtk_use_stub_c_sources)
 @USE_WIN32_TRUE@am__append_16 = -lole32 -lgdi32 -lcomdlg32 -lwinspool -lcomctl32
 @USE_WIN32_TRUE@am__append_17 = -Wl,-luuid
-@USE_WIN32_TRUE@am__append_18 = $(gtk_def) $(gtk_win32_res) $(deps)
+@USE_WIN32_TRUE@am__append_18 = $(gtk_win32_res) $(deps)
 
 # And include the regular ones:
 @HAVE_INTROSPECTION_TRUE@@USE_QUARTZ_TRUE@am__append_19 = \
 @HAVE_INTROSPECTION_TRUE@@USE_QUARTZ_TRUE@    gtkclipboard.c \
 @HAVE_INTROSPECTION_TRUE@@USE_QUARTZ_TRUE@    gtkdnd.c
 
@@ -1170,13 +1170,13 @@
 	$(GDK_HIDDEN_VISIBILITY_CFLAGS)			\
 	$(GTK_DEP_CFLAGS)				\
 	$(INCLUDED_IMMODULE_DEFINE)
 
 @PLATFORM_WIN32_TRUE@no_undefined = -no-undefined
 @OS_WIN32_TRUE@gtk_def = gtk.def
-@OS_WIN32_TRUE@gtk_win32_symbols = -export-symbols $(srcdir)/gtk.def
+@OS_WIN32_TRUE@gtk_win32_symbols =
 @OS_WIN32_TRUE@gtk_win32_res = gtk-win32-res.o
 @OS_WIN32_TRUE@gtk_win32_res_ldflag = -Wl,gtk-win32-res.o
 @MS_LIB_AVAILABLE_TRUE@noinst_DATA = gtk-win32-$(GTK_API_VERSION).lib
 libgtkincludedir = $(includedir)/gtk-3.0/gtk
 libadd = $(top_builddir)/gdk/libgdk-3.la \
 	$(top_builddir)/gtk/inspector/libgtkinspector.la \
@@ -6743,19 +6743,19 @@
 		esac; \
 	done >libgtk.vs10.sourcefiles.filters
 	$(CPP) -P - <$(top_srcdir)/build/win32/vs10/gtk.vcxproj.filtersin >$@
 	rm libgtk.vs10.sourcefiles.filters
 
 # Install a RC file for the default GTK+ theme, and key themes
-install-data-local: install-ms-lib install-def-file install-mac-key-theme
+install-data-local: install-ms-lib install-mac-key-theme
 	$(MKDIR_P) $(DESTDIR)$(datadir)/themes/Default/gtk-3.0
 	$(INSTALL_DATA) $(srcdir)/gtk-keys.css.default $(DESTDIR)$(datadir)/themes/Default/gtk-3.0/gtk-keys.css
 	$(MKDIR_P) $(DESTDIR)$(datadir)/themes/Emacs/gtk-3.0
 	$(INSTALL_DATA) $(srcdir)/gtk-keys.css.emacs $(DESTDIR)$(datadir)/themes/Emacs/gtk-3.0/gtk-keys.css
 
-uninstall-local: uninstall-ms-lib uninstall-def-file uninstall-mac-key-theme
+uninstall-local: uninstall-ms-lib uninstall-mac-key-theme
 	rm -f $(DESTDIR)$(datadir)/themes/Raleigh/gtk-3.0/gtk.css
 	rm -f $(DESTDIR)$(datadir)/themes/Default/gtk-3.0/gtk-keys.css
 	rm -f $(DESTDIR)$(datadir)/themes/Emacs/gtk-3.0/gtk-keys.css
 
 # if srcdir!=builddir, clean out maintainer-clean files from builddir
 # this allows dist to pass.

--- gtk+-3.13.3/gdk/win32/gdkevents-win32.c
+++ gtk+-3.13.3/gdk/win32/gdkevents-win32.c
@@ -1525,17 +1525,16 @@
       DeleteObject (hrgn);
       return;
     }
 
   hdc = BeginPaint (msg->hwnd, &paintstruct);
 
-  GDK_NOTE (EVENTS, g_print (" %s %s dc %p%s",
+  GDK_NOTE (EVENTS, g_print (" %s %s dc %p",
 			     _gdk_win32_rect_to_string (&paintstruct.rcPaint),
 			     (paintstruct.fErase ? "erase" : ""),
-			     hdc,
-			     (return_exposes ? " return_exposes" : "")));
+			     hdc));
 
   EndPaint (msg->hwnd, &paintstruct);
 
   if ((paintstruct.rcPaint.right == paintstruct.rcPaint.left) ||
       (paintstruct.rcPaint.bottom == paintstruct.rcPaint.top))
     {

